iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/ThreeWindingsTransformerModification.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.modification;
+
+import com.powsybl.commons.report.ReportNode;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.modification.topology.NamingStrategy;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.ThreeWindingsTransformer;
+
+import java.util.Objects;
+
+/**
+ * @author Luma Zamarreño {@literal <zamarrenolm at aia.es>}
+ * @author José Antonio Marqués {@literal <marquesja at aia.es>}
+ */
+public class ThreeWindingsTransformerModification extends AbstractNetworkModification {
+
+    private final String transformerId;
+    private final Double ratedU0;
+
+    public ThreeWindingsTransformerModification(String transformerId, Double ratedU0) {
```

Comment:
I am not sure with need a `Double` here.

Reply:
Better double as only one attribute can be modified. Changed.